tools/github-action/pre.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+RUNNER_HOME=/home/runner/work/_temp/_github_home
```

Comment:
we need to keep an eye out whether this ever changes

Reply:
This is true, but there's nothing we can do until it does change.